pvlib/test/test_singlediode_methods.py

```diff
@@ -0,0 +1,172 @@
+"""
+testing way faster single-diode methods using JW Bishop 1988
```

Comment:
Delete 'way faster' I know I'm squelching all the fun out of this

Reply:
✔️